tests/Avalonia.Base.UnitTests/AvaloniaObjectTests_Metadata.cs

```diff
@@ -92,7 +92,7 @@ private class Class3 : AvaloniaObject
             public static readonly DirectProperty<Class3, string> DirectProperty =
                 Class1.DirectProperty.AddOwner<Class3>(o => o.Direct, unsetValue: "baz");
 
-            private string _direct;
+            private string _direct = default;
```

Comment:
See above.

Reply:
AvaloniaObjectTests_Metadata.cs(95,28): warning CS0649: Field 'AvaloniaObjectTests_Metadata.Class3._direct' is never assigned to